.github/workflows/prs-lint-build-package.yml

```diff
@@ -15,6 +15,10 @@ jobs:
       uses: actions/setup-node@v1
       with:
        node-version: 18
+    - name: Setup Python 3.11
+      uses: actions/setup-python@v4
+      with:
+          python-version: '3.11.x'
```

Comment:
CI seems to have updated to `3.12`. node-sass is [deprecated](https://github.com/sass/node-sass/?tab=readme-ov-file#node-sass) and was breaking CI lint because of this issue for one of its dependencies: https://github.com/nodejs/node-gyp/issues/2869
I'll make a new PR to get `sass` working, this is just the least destructive way to make the PR work in the meantime.

Reply:
I suspect this too is related to `macos-latest` now being arm64.

Reply:
@kraenhansen I think it's related in sense that the `arm64` might be shipping with a more updated Python version by default but the error I was getting was specifically related to distuitls not being in Python 3.12, as seen here: https://github.com/nodejs/node-gyp/issues/2869 rather than binaries

Reply:
When installing `node-sass` on the x86 MacOS runner, it doesn't need Python at all, because it downloads a prebuilt binary instead of building from source (requiring Python when running `node-gyp`).

Reply:
ah, interesting, I assumed it always depended on Python, good to know!